Web/Presenters/GroupPresenter.php

```diff
@@ -111,7 +126,27 @@ function renderModifyAdmin(int $id): void
         if(!$club->canBeModifiedBy($this->user->identity ?? NULL) && $club->getOwner()->getId() !== $user->getId())
             $this->flashFail("err", "Ошибка доступа", "У вас недостаточно прав, чтобы изменять этот ресурс.");
 
-        if($removeComment) {
+        if($hidden !== null) {
```

Comment:
OpenVK codestyle uses `!is_null(x)` instead of `x !== null`

Reply:
Done